settings.yaml

```
@@ -30,6 +30,7 @@ local:
   llm_hf_repo_id: TheBloke/Mistral-7B-Instruct-v0.1-GGUF
   llm_hf_model_file: mistral-7b-instruct-v0.1.Q4_K_M.gguf
   embedding_hf_model_name: BAAI/bge-small-en-v1.5
+  llm_hf_resume_download: True
```

Preferring json style boolean (even though `True` is [supported](https://yaml.org/type/bool.html)) as it stay consistent with the `false` in this file.

```suggestion
  llm_hf_resume_download: true
```

---

The option has been removed from settings.